src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -121,7 +121,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'YOUR_INFORMAT
 
       </div>
 
-      <div class="row" ng-if="$ctrl.donorDetails['donor-type'] === 'Household'">
+      <div class="row" ng-if="$ctrl.donorDetails['donor-type'] === 'Household' && $ctrl.useV3 !== 'true'">
```

Comment:
The merge with staging and all the `hideSpouseDetails` config didn't quite go right.